fmn/consumer/backends/mail.py

```diff
@@ -148,8 +148,17 @@ def _format_line(msg):
 
         n = len(queued_messages)
         subject = u"Fedora Notifications Digest (%i updates)" % n
+        summary = u"Digest summary:\n"
```

Comment:
Sorry, I do not understand the request.
At the moment the patch creates a "summary" text and prepend it to the usual digest content if the digest is set to Verbose full, like:
`Digest summary:
1.  First message header
2. Second message header
-----------------------------------------------
<usual digest content with Verbose = Full>
`

If the digest is set to Verbose = false it does nothing (it creates the "summary" text but doesn't prepend to email content.

Do you mean you want to prepend only the "Digest Summary" text to the usual content if Verbose = false? Like
`Digest summary:
<usual digest content with Verbose = False>
`